ui/App.tsx

```diff
@@ -34,11 +34,14 @@ import Sources from "./pages/v2/Sources";
 
 const queryClient = new QueryClient();
 
-function withName(Cmp) {
+function withParams(Cmp) {
```

Comment:
That name feels a bit too broad, technically this function handles only two `params` values.
I'm ok with that and I have no better suggestion.

Reply:
Actually, now I understand what this function does, it can be `withSeachPrams` to be more specific.

Reply:
sound good, will change 👍 
at some point it will also pass in the namespace, which is currently defaulted/hardcoded to "flux-system"